content/beginner/190_fsx_lustre/launching-fsx.md

```diff
@@ -0,0 +1,154 @@
+
+1. Create an AWS Identity and Access Management OIDC provider and associate it with your cluster.
+    ```
+    eksctl utils associate-iam-oidc-provider \
+        --region $AWS_REGION \
+        --cluster $CLUSTER_NAME \
+        --approve
+    ```
+
+    - Create an IAM policy and service account that allows the driver to make calls to AWS APIs on your behalf.
+
+        ```
+        cat > fsx-csi-driver.json
+        ```
+        Copy the following text and save it to a file named fsx-csi-driver.json.
+
+        ```
+    "Version":"2012-10-17",
+        "Statement":[
+            {
+                "Effect":"Allow",
+                "Action":[
+                    "iam:CreateServiceLinkedRole",
+                    "iam:AttachRolePolicy",
+                    "iam:PutRolePolicy"
+                ],
+                "Resource":"arn:aws:iam::*:role/aws-service-role/s3.data-source.lustre.fsx.amazonaws.com/*"
+            },
+            {
+                "Action":"iam:CreateServiceLinkedRole",
+                "Effect":"Allow",
+                "Resource":"*",
+                "Condition":{
+                    "StringLike":{
+                    "iam:AWSServiceName":[
+                        "fsx.amazonaws.com"
+                    ]
+                    }
+                }
+            },
+            {
+                "Effect":"Allow",
+                "Action":[
+                    "s3:ListBucket",
+                    "fsx:CreateFileSystem",
+                    "fsx:DeleteFileSystem",
+                    "fsx:DescribeFileSystems"
+                ],
+                "Resource":[
+                    "*"
+                ]
+            }
+        ]
+        }
+        ```
+
+2. Create the policy.
+```
+    aws iam create-policy \
+        --policy-name Amazon_FSx_Lustre_CSI_Driver \
+        --policy-document file://fsx-csi-driver.json
+```
+    *Take note of the policy Amazon Resource Name (ARN) that is returned.*
+
+3. Create a Kubernetes service account for the driver and attach the policy to the service account. Replacing the ARN of the policy with the ARN returned in the previous step.
+    ```
+    eksctl create iamserviceaccount \
+        --region $AWS_REGION \
+        --name fsx-csi-controller-sa \
+        --namespace kube-system \
+        --cluster $CLUSTER_NAME \
+        --attach-policy-arn arn:aws:iam::<111122223333:policy/Amazon_FSx_Lustre_CSI_Driver> \
+        --approve
+    ```
+
+    **Output:**
+
+    You'll see several lines of output as the service account is created. The last line of output is similar to the following example line.
+
+    ```
+    [ℹ]  created serviceaccount "kube-system/fsx-csi-controller-sa"
+    ```
+
+    Note the name of the AWS CloudFormation stack that was deployed. In the example output above, the stack is named eksctl-prod-addon-iamserviceaccount-kube-system-fsx-csi-controller-sa.
+
+4. Note the Role ARN for the role that was created.
```

Comment:
you can do step 4 from the cli:

export ROLE_ARN=$(aws cloudformation describe-stacks --stack-name eksctl-eksworkshop-eksctl-addon-iamserviceaccount-kube-system-fsx-csi-controller-sa --query "Stacks[0].Outputs[0].OutputValue")

Reply:
hope the order of the output variables are the same hehe